spec/spec_helper.rb

```diff
@@ -112,7 +112,10 @@ def only(*whitelist)
   end unless method_defined?(:only)
 end
 
-# set default time zone in TZ environment variable
-# which will be used to set session time zone
-ENV['TZ'] ||= 'Europe/Riga'
-# ENV['TZ'] ||= 'UTC'
+# Set Oracle Database session timezone
+# this setting needs to be in place to make sure that Oracle properly represents
+# the timestamp format on your local machine
+# Do not use 'OS_TZ' value or fixed offset values like '05:00', for daylight saving-enabled timezones
+# The setting cannot be derived directly from Windows operating system or ruby, as on the Windows machines,
+# timezone names are non-standard ones and are subject to translations to different languages.
+ENV['ORA_SDTZ'] ||= 'Europe/London'
```

Comment:
@jgebal - do you mean on Windows if you don't set ORA_SDTZ (or set it to OS_TZ), then `select sessiontimezone from dual` is not correct representation of your local time zone?

Right now on my environment some tests started failing... Maybe we should leave ORA_SDTZ unset by default (or set to OS_TZ) and override it just in case of need (broken OS or something). Could be via pulling some optional .gitignore-d file.